extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -258,3 +258,30 @@ export async function getSubscriptions(appContext: AppContext, account?: azdata.
 	}));
 	return result;
 }
+
+export async function getSelectedSubscriptions(appContext: AppContext, account?: azdata.Account, ignoreErrors: boolean = false): Promise<GetSubscriptionsResult> {
```

Comment:
I would rather this is just a parameter on getSubscriptions, filteredOnly or something. No reason to duplicate all this logic.

Reply:
making it a parameter in the api, doing it here would be too convulated as selectedSubscriptions do not need token logic at all.